### PR TITLE
[Docs] Add section about creating secrets in project page and English editing

### PR DESCRIPTION
**Pull request by jillnogold**
https://jira.iguazeng.com/browse/ML-1051

### CODE REVIEW

File: docs/secrets.md [hunk 1]
```diff
@@ -43,10 +42,10 @@ def test_function(context, db_name):
     ...
 ```
 
-The {py:func}`~mlrun.model.RunTemplate.with_secrets()` function tells MLRun what secrets the executed code will need 
-access to. The MLRun framework prepares the needed infrastructure to make these secrets available to the runtime, 
+The {py:func}`~mlrun.model.RunTemplate.with_secrets()` function tells MLRun what secrets the executed code needs to 
+access. The MLRun framework prepares the needed infrastructure to make these secrets available to the runtime, 
 and passes information about them to the execution framework by specifying those secrets in the spec of the runtime. 
-For example, if running a kubernetes job, the secret keys will be noted in the generated pod's spec.
+For example, if running a kubernetes job, the secret keys aree noted in the generated pod's spec.
```
Comment (theSaarco) [inline docs/secrets.md:48]:
```suggestion
For example, if running a kubernetes job, the secret keys are noted in the generated pod's spec.
```

**Review by theSaarco — CHANGES_REQUESTED** [review]
Looks good - there's an unfinished sentence there, plus minor typos.

File: docs/secrets.md [hunk 2]
```diff
@@ -135,36 +134,40 @@ action, which normally should only be executed once. After the secrets are popul
...
-The {py:class}`~mlrun.db.httpdb.HTTPRunDB` API does not allow the user to observe secret values, but it does allow 
+The {py:class}`~mlrun.db.httpdb.HTTPRunDB` API does not allow the user to see the secret values, but it does allow 
 users to see the keys that belong to a given project, assuming the user has permissions on that specific project. 
 See the {py:class}`~mlrun.db.httpdb.HTTPRunDB` class documentation for additional details.
 
 When MLRun is executed in the Iguazio platform, the secret management APIs are protected by the platform such
 that only users with permissions to access and modify a specific project can alter its secrets.
 
+##### Creating secrets in the Projects page
+You can also create secrets in the Projects page by clicking the
```
[the remark this answers is not in the document]
Comment (jillnogold) [reply docs/secrets.md]: added the missing text

**Review by theSaarco — APPROVED** [review]
Looks good.

**Pull request merged**